src/Maybe/find.spec.js

```diff
@@ -0,0 +1,47 @@
+const test = require('tape')
+
+const isFunction = require('../core/isFunction')
+
+const find = require('./find')
+
+const helpers = require('../test/helpers')
+
+const List = require('../core/List')
+const { fromArray } = List
+
+const bindFunc = helpers.bindFunc
+
+test('find is protected', t => {
+  const fn = bindFunc(fn => find(fn, []))
+  const err = /find: Function required for first argument/
+
+  t.throws(fn(undefined), err, 'throws if foldable is undefined')
+  t.throws(fn(null), err, 'throws if foldable is null')
+  t.throws(fn(0), err, 'throws if foldable is falsey number')
+  t.throws(fn(1), err, 'throws if foldable is truthy number')
+  t.throws(fn(NaN), err, 'throws if foldable is NaN')
+  t.throws(fn(''), err, 'throws if foldable is falsey string')
+  t.throws(fn('string'), err, 'throws if foldable is truthy string')
+  t.throws(fn(false), err, 'throws if arg is false')
+  t.throws(fn(true), err, 'throws if arg is true')
+  t.throws(fn({}), err, 'throws if arg is empty POJO')
+  t.throws(fn({ hi: 'there' }), err, 'throws if arg is non-empty POJO')
+
+  t.end()
+})
+
+test('find works as expected', t => {
```

Comment:
We should include a section to get some coverage around using a `Pred` datatype.

Reply:
Still need to add specs using the `Pred` datatype.